crates/sui-framework/packages/bridge/sources/bridge.move

```diff
@@ -183,42 +183,37 @@ module bridge::bridge {
         let inner = load_inner_mut(self);
         let key = message::key(&message);
 
-        // TODO: use borrow mut
+        // verify signatures
+        committee::verify_signatures(&inner.committee, message, signatures);
 
```

Comment:
if sigs are already approved or token already claimed, do we want to skip this step to save gas?